gobblin-example/src/main/java/org/apache/gobblin/example/githubjsontoparquet/GithubDataEventTypesPartitioner.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.gobblin.example.githubjsontoparquet;
+
+import org.apache.avro.Schema;
+import org.apache.avro.SchemaBuilder;
+import org.apache.avro.generic.GenericData;
+import org.apache.avro.generic.GenericRecord;
+import org.apache.gobblin.writer.partitioner.WriterPartitioner;
+
+
+/**
+ * Partitioner for github json records based on 'PARTITION_KEY' key.
+ * @author tilakpatidar
+ */
+public class GithubDataEventTypesPartitioner implements WriterPartitioner<GenericRecord> {
+
+  private static final String SCHEMA_STRING = "schemaString";
```

Comment:
@tilakpatidar What do you think of this comment?

Reply:
@zxcware I realised that the partitioning was not happening when running the example. 
The first problem was using ```WriterPartitioner<GenericRecord>``` instead of ```WriterPartitioner<ParquetGroup>```. 

However, if you see the code in [WriterPartitioner](https://github.com/apache/incubator-gobblin/blob/master/gobblin-core-base/src/main/java/org/apache/gobblin/writer/partitioner/WriterPartitioner.java) and this method in it.

```java
  public GenericRecord partitionForRecord(D record);
```
You can see that it accepts any type of record but the output should in GenericRecord which is Avro, not ParquetGroup which is parquet. This partitionForRecord method is being used in [PartitionedDataWriter](https://github.com/apache/incubator-gobblin/blob/master/gobblin-core/src/main/java/org/apache/gobblin/writer/PartitionedDataWriter.java) whose implementation is tightly coupled with Avro.

Maybe, a different PR is needed for supporting writing any file format using [PartitionedDataWriter](https://github.com/apache/incubator-gobblin/blob/master/gobblin-core/src/main/java/org/apache/gobblin/writer/PartitionedDataWriter.java). 
For now, I am removing the partitioner from this example.
What are your thoughts on this?

Reply:
@zxcware Ignore the above the comment.